Added optional check if font files exists

robinvdvleuten opened the pull request:
Hi there,

I've added an optional check if the font files actually exists. It's turned off by default so anyone who updates has the expected behaviour. Hopefully you'll like the idea.

Cheers,
Robin

seaneking commented:
Looks great! Few minor things: 

- I think an option of `checkPath` makes more sense than `checkMissing`? 

- For the sake of simplicity I'd say if `checkMissing` === true just warn on missing fonts and skip them, rather than making that configurable through `ignoreMissing`. 

- We'll need to document this in the README before merge.

robinvdvleuten commented:
@seaneking I've renamed `checkMissing` to `checkPath` and removed `ignoreMissing`. This makes indeed more sense and removes an extra "unneeded" option. It now just logs a warning and skips the file in the output. I've also updated the readme with the new option.

seaneking commented:
:+1: LGTM. Thanks!
